src/OneExplorer/OneExplorer.ts

```diff
@@ -501,12 +500,20 @@ input_path=${modelName}.${extName}
 
 export class OneExplorer {
   // TODO Support multi-root workspace
-  public workspaceRoot: vscode.Uri = vscode.Uri.file(obtainWorkspaceRoot());
+  public workspaceRoot: vscode.Uri|undefined;
   public treeView: vscode.TreeView<OneNode|undefined>|undefined;
 
   constructor(context: vscode.ExtensionContext) {
-    // NOTE: Fix `obtainWorksapceRoot` if non-null assertion is false
-    const oneTreeDataProvider = new OneTreeDataProvider(this.workspaceRoot!);
+    try {
+      this.workspaceRoot = vscode.Uri.file(obtainWorkspaceRoot());
+    } catch (e: unknown) {
+      let errmsg = 'Workspace has not been defined yet.';
+      if (e instanceof Error) {
```

Comment:
Q. What is normally given as e?
Under what situation does errmsg get printed..?

Reply:
I referenced few articles, and [here](https://velog.io/@shroad1802/%ED%83%80%EC%9E%85%EC%8A%A4%ED%81%AC%EB%A6%BD%ED%8A%B8%EC%99%80-catch%EC%A0%88-%EC%97%90%EB%9F%AC-%EC%B2%98%EB%A6%AC) is one of them. Since typescript can throw any types as error, it's usual to use such error handling method.

Reply:
Using `try/catch` in the constructor is not recommended because it cannot be called again. However, in our use case, we do not need to invoke this constructor again, and when user opens the workspace, it restarts and invokes this function again. So I agree this PR. :)

(optional) How about showing the error message to user? It only prints the error on output window.

Reply:
Now I understood. Then, how can we log the error message? How do you think about this approach?

```ts
catch (e: unknown){
  if (e instance of Error) {
    Logger.error('OneExplorer', e.message);
  } else {
    throw e
  }
}
```